aip/0217.md

```diff
@@ -66,10 +72,17 @@ request with an error.
 When paginating over a list, it is likely that the service will not know that
 there are unreachable locations or resources initially.
 
-- The response **may** provide the unreachable location or resource in the
-  `unreachable` field on the page where it is initially encountered.
 - The response **must** provide any outstanding unreachable locations or
-  resources in the `unreachable` field on the final page.
+  resources in the `unreachable` field on the _final_ page.
```

Comment:
It would probably be clearer to write "on pages following the final page that contains resource". There's an implication in "the final page" that there's only one such page.